dev_docs/cxg.md

```diff
@@ -0,0 +1,162 @@
+
+```
+{
+    "louvain_colors": {
+        "0": "#FFFFFF",
+        "1": "#000000"
+    }
+}
+```
+
+## Corpora Schema Encoding
+
+The [Corpora schema](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema.md) and [Corpora AnnData encoding](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md) define a set of metadata and encoding conventions for annotated matrices.  When a Corpora dataset is encoded as a CXG, the following shall apply.
+
+### Corpora metadata property
+
+A CXG containing a Corpora dataset will contain a property in the __cxg_group_metadata__ field named `corpora`. The value will be a JSON encoded string, which in turn contains all properties defined in the [Corpora AnnData uns](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md#uns) container. For example:
+
+```
+{
+    "corpora": {
+        "version": {
+            "corpora_schema_version": "1.0.0",
+            "corpora_encoding_version": "0.1.0",
+        }
+    }
+}
+```
+
+The `corpora` metadata, if present, MUST contain the version information. Optionality of other values in this object will follow the specifications set forth in the relevant Corpora schema specification (ie, optional fields are optional, required are present, etc), with the following changes:
+* the contents of `corpora_encoding_version` MUST be identical to the `cxg_version`, as this field is defined as the current object encoding version, *NOT* the source data encoding version.
+* the entire encoding will be JSON, rather than a hybrid Python/JSON encoding, but will otherwise follow the data structure defined by the AnnData Corpora encoding.
+* the `<obs_column>_colors` will be omitted in favor of `cxg_category_colors`
+
+### Other Corpora fields
+
+All other Corpora schema fields will be encoded into a CXG using the conventions defined in the [Corpora Schema AnnData Implementation](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema_h5ad_implementation.md). For example, fields in `AnnData.obs` will be encoded in the CXG `obs`array as defined [above](#obs-and-var).
+
+### Compatibility with CXG 0.1.0
+
+For backwards compatibility and continuity with CXG version 0.1.0, the following MUST be implemented.
+
+#### Presentation Hints
+* The [Corpora `title`](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema.md#presentation-metadata) value MUST be saved in the `cxg_properties.title` field.
+* The [Corpora `color_map`](https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/schema/corpora_schema.md#presentation-hints) MUST be saved in the `cxg_category_colors` field.
```

Comment:
I think is is listed as optional in the corpora schema.  "Submitters can include a field called {field}_colors..."

Reply:
correct, but when it is present, it must be int that field, rather than in the corpora field.

I'll revise the wording to make it more obvious what I'm trying to say.